src/main/java/Duke.java

```diff
@@ -1,10 +1,89 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static final String EXIT_MESSAGE = "bye";
+    public static final String PRINT_MESSAGE = "list";
+    public static final String MARK_MESSAGE = "mark";
+    public static final String UNMARK_MESSAGE = "unmark";
+    public static final String TODO_MESSAGE = "todo";
+    public static final String DEADLINE_MESSAGE = "deadline";
+    public static final String EVENT_MESSAGE = "event";
+
+    private static void printWelcomeMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo + "Hello! I'm Duke\nWhat can I do for you?\n");
+    }
+
+    public static void printList(Task[] list, int listCounter) {
+        for (int i = 0; i < listCounter; i++) {
+            int listIndex = i + 1;
+            System.out.println(listIndex + "." + list[i]);
+        }
+    }
+
+    public static int getTaskIndex(String userInput) {
+        String taskNumber = userInput.split(" ")[1];
+        int taskIndex = Integer.parseInt(taskNumber) - 1;
+        return taskIndex;
+    }
+
+    public static void addList(Task[] list, int listCounter, String userInput) {
```

Comment:
Consider adding documentation to each function.

Reply:
good observation! thanks!